st2common/st2common/runners/paramiko_ssh.py

```diff
@@ -455,6 +456,12 @@ def close(self):
 
         self.client.close()
 
+        if self.socket:
+            self.logger.debug('Closing proxycommand socket connection')
+            #https://github.com/paramiko/paramiko/issues/789  Avoid zombie ssh processes
```

Comment:
This is failing the ci check:
https://travis-ci.org/github/StackStorm/st2/jobs/660286343
```suggestion
            # https://github.com/paramiko/paramiko/issues/789  Avoid zombie ssh processes
```